### PR TITLE
Add micrometer to issue auto-labelling configuration

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: .github/autoissuelabeler.yml [hunk 1]
```diff
@@ -79,6 +79,10 @@
   title: "(?i).*metrics.*"
   description: "(?i).*metrics.*"
   notify: [jmartisk]
+- labels: [area/metrics]
+  title: "(?i).*micrometer.*"
+  description: "(?i).*micrometer.*"
+  notify: [ebullient]
```
Comment (machi1990) [inline .github/autoissuelabeler.yml:85]: And @jmartisk too?

**Review by machi1990 — APPROVED** [review]
Added one comment, otherwise LGTM.

File: .github/autoissuelabeler.yml (hunk 1 above)
Comment (geoand) [reply .github/autoissuelabeler.yml:85]: Given that Micrometer is mostly done @ebullient, I think that she can ping Jan when needed

**Comment by ebullient** [comment]
Thanks, Georgios!

**Pull request merged**